Automattic/wp-calypso
# Hosting Metrics: Fix formatting of dates on the line charts

Related to #https://github.com/Automattic/wp-calypso/pull/80382

## Proposed Changes

This PR adds a date and time formatting for the legend. If the time range that is selected is 6 hours or 24 hours, the legend will display only time:

<img width="1022" alt="Screenshot 2023-08-11 at 1 13 09 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/a4485cd2-bb3c-48f2-8089-a710bc4be269">

If the range is 3 or 7 days, the legend will display the times and dates:

<img width="1006" alt="Screenshot 2023-08-11 at 1 13 55 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/136fae16-a9c9-44a6-931a-57a58398d800">

## Testing Instructions

* Navigate to Calypso Live link
* Append `site-monitoring/{yourAtomicSite}?page=metrics` on your Atomic site
* Observe the line chart with 24 hours or 6 hours and confirm that the legend shows the times
* Switch to 3 days and 7 days and confirm that the legend shows times and dates

**Notes:**
* We are aware that this is slightly different from the suggested design and we can improve the times' display in the future iterations 

## Pre-merge Checklist
